Test-extended.functional linux_x86 testMemoryLeaks IdleGCManager.cpp:40

External builds didn't work last night, but internal builds on linux_x86 are showing the following failure.
This isn't a new failure, not sure why it doesn't show up in the external builds.

VmArgumentTests_0
```
FAILED: testMemoryLeaks
java.lang.AssertionError: -Xcheck:memory failed: stderr = WARNING: 1 unfreed blocks remaining at shutdown!
WARNING: 0 blocks were ignored per ignoredCallsite parameter
port_shutdown_library describing block at 00007F41A0277410 (header at 00007F41A0277210):
Wrapped block size is 32, allocation number is 1108
Block was allocated by IdleGCManager.cpp:40
First 32 bytes:
00007F41A0277410: 10 00 00 00 00 00 00 00 00 00 00 00 27 A0 E7 E7  ............'...
00007F41A0277420: 90 63 45 99 41 7F 00 00 40 DF 05 A0 41 7F 00 00  .cE.A...@...A...
Memory checker statistics:
Total blocks allocated = 10902 ( = most recent allocationNumber)
Total blocks freed = 10901
Total bytes allocated = 76767336
Total bytes freed = 76767304
Total unknown blocks ignored = 0
High water blocks allocated = 7140
High water bytes allocated = 74168169
Largest block ever allocated = size 10924344, allocation number 315
Failed allocation attempts = 0
1 allocated blocks totaling 32 bytes were not freed before shutdown!
```

--- a/runtime/gc_base/GCExtensions.cpp
+++ b/runtime/gc_base/GCExtensions.cpp
@@ -33,6 +33,9 @@
 
 #include "EnvironmentBase.hpp"
 #include "Forge.hpp"
+#if defined(J9VM_GC_IDLE_HEAP_MANAGER)
+ #include  "IdleGCManager.hpp"
+#endif /* defined(J9VM_GC_IDLE_HEAP_MANAGER) */
 #include "MemorySubSpace.hpp"
 #include "ObjectModel.hpp"
 #include "ReferenceChainWalkerMarkMap.hpp"
@@ -173,6 +176,13 @@ MM_GCExtensions::tearDown(MM_EnvironmentBase *env)
 		(*tmpHookInterface)->J9HookShutdownInterface(tmpHookInterface);
 		*tmpHookInterface = NULL; /* avoid issues with double teardowns */
 	}
+
+#if defined(J9VM_GC_IDLE_HEAP_MANAGER)
+	if (NULL != idleGCManager) {
+		idleGCManager->kill(env);
+		idleGCManager = NULL;
+	}
+#endif /* defined(J9VM_GC_IDLE_HEAP_MANAGER) */
 
 	MM_GCExtensionsBase::tearDown(env);
 }

